.gitignore

```diff
@@ -37,3 +37,5 @@ package-lock.json
 
 .env
 /nbproject/
+
+junit.xml
```

Comment:
Was this a purposeful change?  Or maybe accidental? Just seems unrelated to the rest of this PR.

Reply:
I added that, junit.xml is generated for an internal test coverage check at Atmire, and I saw a couple of instances of devs accidentally committing it in contribution branches and then having to remove it. It can't hurt to add it to gitignore imo.